applications/Defibox_vault.md

```diff
@@ -0,0 +1,234 @@
+# EOS Network Foundation Grant Proposal
+
+- **Project Name:** Vault Protocol
+- **Team Name:** Defibox
+- **EOS Payment Address:** liquid.defi
+- **[Level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels):** 3
+- **Pomelo Grant(s):** https://pomelo.io/grants/vault
+- **Project is Open-Source:** Yes*
+- **Project was part of Token sale:** No
+- **Repository where Project resides:** https://github.com/DefiboxTeam/DefiboxVault-interface
```

Comment:
The project repo referenced here is empty

I'm assuming it should be the following URL: https://github.com/defiboxswap/DefiboxVault-interface

Reply:
Currently, there is no license in this repo. Which license are you planning on using?

Reply:
Appropriate license has been added.
https://github.com/defiboxswap/DefiboxVault-interface/blob/main/LICENSE

Reply:
Question: The repo only contains header files. There is no related code. How is the license even useful when only `hpp` will be made public?

Reply:
@Mattijah If you review the **Open source - Timelines** it indicates the Stage 2/3 will include the full code to be released open source.

For security reasons, the *.cpp is not made available in the first stage, as the vault contains an excess of 9M EOS.